libparsec/crates/testbed/src/template/events.rs

```diff
@@ -85,6 +85,7 @@ macro_rules! impl_certificates_meth_for_single_certificate {
             ) -> impl Iterator<Item = TestbedTemplateEventCertificate> + 'c {
                 std::iter::once(()).map(move |_| {
                     let mut guard = self.cache.lock().expect("Mutex is poisoned");
+                    #[allow(clippy::redundant_closure_call)]
```

Comment:
Can we fix it ?

Reply:
Instead of

```rust
#[allow(clippy::redundant_closure_call)]
let populate = || $populate(self, template);
```

I can do

```rust
let populate_fn = $populate;
let populate = || populate_fn(self, template);
```

Reply:
FYI, the warning is raise because the expension would look like something:

```rust
let populate || (|foo: u32, bar: String| format!("{bar}-{foo}"))(42,"anwser".into());
```

Reply:
Can you fix it ?

Reply:
Yes, I just didn't have time :sweat_smile:

Reply:
done